iphone/CoreApi/CoreApi/Formatting/AltitudeFormatter.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+// TODO: Replace this class with a core's implementation
```

Comment:
Did you check platform::Distance::FormatAltitude ?

Reply:
fixed